modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Checkbox/Checkbox.es.js

```diff
@@ -261,4 +262,145 @@ describe('Field Checkbox', () => {
 			expect(ddmInfo).not.toBeNull();
 		});
 	});
+
+	describe('Boolean Field', () => {
+		it('Has predefinedValue false', () => {
```

Comment:
- Start test description with lower case, cause it is kind of append to the describe. So, when this test is execute it title becomes something like this: "Boolean Field Has predefinedValue false". 
- Give meaningful descriptions, in order to give the proper understand to the people that would facing failures running this the tests 
```suggestion
		it('has predefined value set', () => {
```